lib/shoryuken/middleware/server/exponential_backoff_retry.rb

```diff
@@ -34,7 +34,7 @@ def get_interval(retry_intervals, attempts)
             retry_intervals[attempts]
           else
             retry_intervals.last
-          end
+          end.to_i
```

Comment:
@felixbuenemann is that what you were asking for in https://github.com/phstc/shoryuken/issues/303#issuecomment-285438722?